src/eventSchemas/vue/searchSchemas.vue

```diff
@@ -1,8 +1,17 @@
 <template>
-    <h1>
-        {{ initialData.Header }}
-    </h1>
     <div id="app">
+        <div class="container button-container" style="justify-content: space-between">
+            <h1>{{ initialData.Header }}</h1>
+            <div>
+                <input
```

Comment:
You can use the `button` tag instead. 
Not sure why any of the webviews ever used `form` or `submit`. There's no form handler to invoke.